src/components/DirectionIcon/DirectionIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { SvgIcon } from '@mui/material';
+
+interface Props {
```

Comment:
interface lepiej nazywaćw ten sposób 
```suggestion
interface DirectionIconProps {
```

w większych projektach, będą sytuacje gdzie w jednym pliku bedziecie mieli wiecej interfaceów wiec trzeba je nazywac dokłądnie

Reply:
Zrobione.